code/modules/customitems/item_defines.dm

```diff
@@ -360,6 +360,75 @@
 		return
 	to_chat(user, "<span class='warning'>You can't modify [target]!</span>")
 
+/obj/item/device/fluff/lighty_plasman_modkit // LightFire53: Ikelos
+	name = "plasmaman suit modkit"
+	desc = "A kit containing nanites that are able to modify the look of a plasmaman suit and helmet without exposing the wearer to hostile environments."
+	icon_state = "modkit"
+	w_class = 2
+	force = 0
+	throwforce = 0
+	var/list/options = list("Blue", "Gold", "Red")
+	var/picked_color = null
+
+/obj/item/device/fluff/lighty_plasman_modkit/afterattack(atom/target, mob/user, proximity)
+	if(!proximity || !ishuman(user) || user.incapacitated())
+		return
+	var/mob/living/carbon/human/H = user
+
+	if(istype(target, /obj/item/clothing/head/helmet/space/eva/plasmaman))
+		if(used & USED_MOD_HELM)
+			to_chat(H, "<span class='notice'>The kit's helmet modifier has already been used.</span>")
+			return
+
+		picked_color = input(H, "Which color would you like to paint [target]?", "Recolor") as null|anything in options
+		var/obj/item/clothing/head/helmet/space/eva/plasmaman/P = target
+
+		switch(picked_color)
```

Comment:
If you're going to make this a pre-initialized list, just make it an associative array linking names to the icon state